engine/table/src/main/java/io/deephaven/engine/table/impl/select/ZonedDateTimeRangeFilter.java

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.engine.table.impl.select;
+
+import io.deephaven.base.verify.Assert;
+import io.deephaven.engine.table.ColumnDefinition;
+import io.deephaven.engine.table.TableDefinition;
+import io.deephaven.engine.table.impl.chunkfilter.ChunkFilter;
+import io.deephaven.engine.rowset.chunkattributes.OrderedRowKeys;
+import io.deephaven.engine.table.ColumnSource;
+import io.deephaven.engine.table.impl.sources.ReinterpretUtils;
+import io.deephaven.chunk.*;
+import io.deephaven.chunk.attributes.Values;
+import io.deephaven.engine.rowset.WritableRowSet;
+import io.deephaven.engine.rowset.RowSet;
+import io.deephaven.time.DateTimeUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.ZonedDateTime;
+
+public class ZonedDateTimeRangeFilter extends LongRangeFilter {
```

Comment:
We should actually explicitly discuss if this is good. ZonedDateTime equals and comparison are not consistent with Instant or epoch nanos.